cni-plugins.yaml

```diff
@@ -105,6 +113,31 @@ subpackages:
       - runs: |
           mkdir -p "${{targets.subpkgdir}}"/opt/cni/bin
           ln -s /usr/bin/${{range.key}} "${{targets.subpkgdir}}"/opt/cni/bin/${{range.key}}
+  - name: "cni-plugins-aws-k8s-compat"
+    description: "Compatibility package for aws-k8s-cni"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/init
+          for f in bandwidth \
```

Comment:
We should place all of these in a var above and loop over that

Reply:
Example with PostgreSQL: https://github.com/wolfi-dev/os/blob/2f02bc0659d420eab283f8553856a450f1ab9b4d/postgresql-16.yaml#L88

Reply:
that's neat, will change it!

Reply:
ended up doing this for the other subpackages too, definitely makes it easier to maintain